shortcircuit-api/src/main/java/com/powsybl/shortcircuit/FeederResult.java

```diff
@@ -26,18 +26,31 @@ public class FeederResult {
      */
     private final double feederThreePhaseCurrent;
 
```

Comment:
I am thinking about making `current`, that contains really detailed values, optional in order to light the API in case of symmetrical computation. The other option could be to have 2 data structures, one `ThreePhaseValue` for assymmetrical calculation and another one `OnePhaseValue` (or any better proposal) for symmetrical computations.

Reply:
See my comment above, having a unique structure but more dynamic to be able to compare balanced/unbalanced results and avoid to store useless information in case of balanced